settings.gradle

```diff
@@ -89,5 +90,17 @@ if (localBuilds.exists()) {
             }
         }
     }
+
+    // Not sure what to use in place of :WordPressUtils here. Leaving it for
+    // now.
```

Comment:
I agree that ignoring this for now makes sense. WPAndroid already has a working capability to build using gb-mobile source by using the `wp.BUILD_GUTENBERG_FROM_SOURCE` flag, so we can focus on getting the binary build working (without breaking the current build-from-source mode). If you get that working you're going to have a lot of new Android BFFs!

After that, we could work on updating our build-from-source to use the new (and better) approach in a separate follow-up PR. With that said, if you want to go ahead and knock them both out, don't let me stop you! 😆